packages/components/src/Combobox/Combobox.test.tsx

```diff
@@ -52,11 +52,6 @@ describe("Combobox", () => {
       expect(screen.getByText("Add Teammate")).toBeInTheDocument();
     });
 
-    it("should close the menu when clicking the activator", async () => {
-      await userEvent.click(screen.getByRole("combobox"));
-      expect(screen.getByTestId(MENU_TEST_ID)).toHaveClass("hidden");
-    });
-
```

Comment:
this test was doing something users cannot do by finding the DOM element and programatically clicking it